wasm/wasm_source/src/vp_validator.rs

```diff
@@ -562,12 +641,13 @@ mod tests {
             tx::ctx()
                 .unbond_tokens(Some(&vp_owner), &validator, unbond_amount)
                 .unwrap();
-            tx::ctx()
-                .change_validator_commission_rate(
-                    &validator,
-                    &Dec::new(6, 2).unwrap(),
-                )
-                .unwrap();
+            // tx::ctx()
+            //     .change_validator_commission_rate(
+            //         &validator,
+            //         &Dec::new(6, 2).unwrap(),
+            //     )
+            //     .unwrap();
+            // tx::ctx().deactivate_validator(&validator).unwrap();
```

Comment:
I'll uncomment this again - the reason it's being rejected is that we're using a different `vp_owner` address from the `validator` address, it should be the same one

Reply:
done in e4329ce944b84518785469ebfade8389e963016e